File: Labor5.py
amount=0
for a in range(1000000, 1200000):
    b=a
    answer=0
    while b!=0:
        temp=b%10
        answer = answer * 10 + temp
        b //= 10
    if a==answer:
        amount+=1
        print(a, end='; ')
print('Amount: '+str(amount))

File: Labor5.cpp
#include <iostream>

int main()
{
	int a;
	long amount = 0;
	for (a = 1000000; a <= 1300000; a++) {
		int b = a;
		int answer = 0;
		while (b)
		{
			int temp = b % 10;
			answer = answer * 10 + temp;
			b /= 10;
		}
		if (a == answer) {
			amount++;
			std::cout << a << "; ";
		}
	}
	std::cout << "Amount: " << amount;
	return 0;
}